src/app/layout.tsx

```diff
@@ -1,32 +1,60 @@
 import "./globals.css";
 
 import type { Metadata } from "next";
-import { Inter as FontSans } from "next/font/google";
+import localFont from "next/font/local";
 
 import { RootLayout } from "#/components/RootLayout";
 import { cn } from "#/utils";
 
-const fontSans = FontSans({
-  subsets: ["latin"],
-  variable: "--font-sans",
-});
-
 export const metadata: Metadata = {
   title: "Create Next App",
   description: "Generated by create next app",
 };
 
+const flechaS = localFont({
```

Comment:
any reason for us to be using the same branding from CoW AMM here? 🤔

Reply:
we are just using for now, just bc it was a theme that was already defined. Maybe CoW team can help us on this.